src/plugins/load.js

```diff
@@ -37,10 +37,11 @@ module.exports = function (Plugins) {
 	};
 
 	Plugins.prepareForBuild = function (callback) {
-		Plugins.cssFiles.length = 0;
-		Plugins.lessFiles.length = 0;
-		Plugins.clientScripts.length = 0;
-		Plugins.acpScripts.length = 0;
+		Plugins.cssFiles = [];
+		Plugins.lessFiles = [];
+		Plugins.clientScripts = [];
+		Plugins.acpScripts = [];
+		Plugins.soundpacks = [];
```

Comment:
I use `.length = 0` as it doesn't create a new array reference. Going forward, remember that in a team environment, you should not change idiosyncrasies in other peoples' code just because you don't agree with them, unless there is a compelling reason to do so. 😉